lms/envs/devstack_docker.py

```diff
@@ -30,8 +30,10 @@
     'ENABLE_COURSEWARE_SEARCH': False,
     'ENABLE_COURSE_DISCOVERY': False,
     'ENABLE_DASHBOARD_SEARCH': False,
-    'SHOW_LANGUAGE_SELECTOR': True,
```

Comment:
Why are we deleting this in one place but not others?
Are you ready to remove/deprecate it?